oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -468,6 +470,11 @@ Next Steps
 Change History
 ==============
 
+2023-05-11
+----------
+
+* Minor update to the ADR template in OEP-19, providing a specific example for status. The new example provides a date format, and also removes the bullet that may be used if copying from the list of examples of status.
```

Comment:
I see the addition but not the removal.  Did something not get checked in?

Reply:
Some were setting status to `- **Accepted** *2023-05-11*`, rather than `**Accepted** *2023-05-11*`, because the hyphen was being copied from the list. This hyphen was not included in the new specific example.